lib/src/provider/read_provider.dart

```diff
@@ -1,5 +1,9 @@
+import 'package:starknet/src/provider/model/get_block_txn_count.dart';
```

Comment:
never import from deep nested path. You should add your file to the `exports` in the `index.dart`